packages/mapper/src/mapper.integration.spec.ts

```diff
@@ -3,11 +3,21 @@ import { CliServiceProvider } from '@mongosh/service-provider-server';
 import Mapper from './mapper';
 import { Collection, Cursor, Database } from '@mongosh/shell-api';
 
+const mongodbRunnerBefore = require('mongodb-runner/mocha/before');
+const mongodbRunnerAfter = require('mongodb-runner/mocha/after');
+
 describe('Mapper (integration)', function() {
-  this.timeout(10000);
+  this.timeout(60000);
+
+  before(function(done) {
+    try {
+      mongodbRunnerBefore({ port: 27018, timeout: 60000 }).call(this, done);
```

Comment:
I wonder if wrapping the mongodb-runner/mocha/before in a `try`/`catch` might hide the problem with this module on windows.